### ISSUE TITLE
New mail-mark-junk icon is too distracting

### ISSUE THREAD

**Issue by tintou** [elementary/icons#755]
After some time testing this icon, It feel that the red is too strong and is very eye-catching when I don't have to mark things as spam that often.

![capture d ecran du 2019-01-17 17 04 46 2x](https://user-images.githubusercontent.com/745535/51331460-18727f80-1a7a-11e9-8b30-f21292a2c271.png)
![capture d ecran du 2019-01-17 17 02 56 2x](https://user-images.githubusercontent.com/745535/51331459-18727f80-1a7a-11e9-850f-ac8e64da9491.png)

**Comment by cassidyjames** [comment]
Yeah, maybe we tone it down a bit. It's pretty intense in-context.
